### PR TITLE
BXMSPROD-32 add-ons module created with all the add-ons in one zip

### PR DESCRIPTION
**Pull request by Ginxo**

### CODE REVIEW

**Comment by mareknovotny** [comment]
@domhanak are you fine with this PR which follows up BXMSPROD-33 and merged PR #941 941

**Comment by mareknovotny** [comment]
@Ginxo can you rebase this PR?

**Comment by mareknovotny** [comment]
Jenkins retest this

**Comment by Ginxo** [comment]
@mareknovotny it's failing in a point which is not affected by this change

**Comment by mareknovotny** [comment]
@Ginxo I think it is related as there is older version reference to parent, we should have there  https://github.com/kiegroup/kie-wb-distributions/blob/master/business-central-parent/pom.xml#L21

File: business-central-parent/add-ons-distribution/pom.xml [hunk 1]
```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>business-central-parent</artifactId>
+        <groupId>org.kie</groupId>
+        <version>7.24.0-SNAPSHOT</version>
```
Comment (mareknovotny) [inline business-central-parent/add-ons-distribution/pom.xml]: We need to have here 7.26.0-SNAPSHOT

Comment (Ginxo) [reply business-central-parent/add-ons-distribution/pom.xml]: Done!

**Comment by mareknovotny** [comment]
not sure the failed test is really related to the changes.

but trying again Jenkins please retest this

**Pull request merged**